diff --git a/build/src/start/server/metro/MetroBundlerDevServer.js b/build/src/start/server/metro/MetroBundlerDevServer.js
index b2fd0875849aba7ff128f0527f6640dfaf24a66e..9e774bb99a913285c2177847ab0b9d067c44da7a 100644
--- a/build/src/start/server/metro/MetroBundlerDevServer.js
+++ b/build/src/start/server/metro/MetroBundlerDevServer.js
@@ -783,7 +783,9 @@ class MetroBundlerDevServer extends _BundlerDevServer.BundlerDevServer {
         };
         // https://github.com/facebook/metro/blob/2405f2f6c37a1b641cc379b9c733b1eff0c1c2a1/packages/metro/src/lib/parseOptionsFromUrl.js#L55-L87
         if (!opts.mainModuleName.startsWith('/') && !_path().default.isAbsolute(opts.mainModuleName)) {
-            opts.mainModuleName = './' + opts.mainModuleName;
+            // FIXME: Hard-coded due to not working as expected in monorepo.
+            opts.mainModuleName = "./example";
+            // opts.mainModuleName = './' + opts.mainModuleName;
         }
         const output = await this.metroLoadModuleContents(opts.mainModuleName, opts, extraOptions);
         return {
