diff --git a/src/next/FileSystem.ts b/src/next/FileSystem.ts
index b020328b0d0a1617b24557ce4f30a39c72817028..a07c842f15ea1feff1790457254a9c2834d16337 100644
--- a/src/next/FileSystem.ts
+++ b/src/next/FileSystem.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import ExpoFileSystem from './ExpoFileSystem';
 import { PathUtilities } from './pathUtilities';
 import { FileSystemReadableStreamSource, FileSystemWritableSink } from './streams';
diff --git a/src/next/pathUtilities/path.ts b/src/next/pathUtilities/path.ts
index 839050564e2984e455f7b7792401103096c3684b..05a4d21340ddb837e539d8e43293009e9a2d06fa 100644
--- a/src/next/pathUtilities/path.ts
+++ b/src/next/pathUtilities/path.ts
@@ -18,7 +18,7 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
-
+// @ts-nocheck
 function isPathSeparator(code: string) {
   return code === '/';
 }
diff --git a/src/next/streams.ts b/src/next/streams.ts
index 26b808b4cd77ef446a3049e6c936d242ff57df1c..4a14efc0209a5e3bbd413c73ecf0802a8020b79d 100644
--- a/src/next/streams.ts
+++ b/src/next/streams.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import type { FileHandle } from './ExpoFileSystem.types';
 
 export class FileSystemReadableStreamSource implements UnderlyingByteSource {
